php/templates/containers.twig

```diff
@@ -92,9 +92,16 @@
                             <input class="button" type="submit" value="Submit" />
                         </form>
```

Comment:
Link to RP docs then

Reply:
aka "AIO is currently in normal mode - that means it cannot be run behind a web server or reverse proxy like apache nginx and else. If you need that, see ... for a different docker run command".

AIO is currently in reverse proxy mode - that means it can be run behind a web server or reverse proxy like apache nginx and else. See and follow docs for more info."